core/amber/src/main/scala/edu/uci/ics/texera/web/service/ExecutionResultService.scala

```diff
@@ -336,10 +358,6 @@ class ExecutionResultService(
       val newInfo: Map[OperatorIdentity, OperatorResultMetadata] = {
         WorkflowExecutionsResource
           .getResultUrisByExecutionId(executionId)
-          .filter(uri => {
-            val (_, _, _, _, _, resourceType) = VFSURIFactory.decodeURI(uri)
-            resourceType != MATERIALIZED_RESULT
-          })
```

Comment:
I know we have unified the storage for both sys-generated materializations and user-requested results. But do we want to show and update all storage to frontend? 

A senario: if we generated a metarialization after build operator of hashjoin, do we show it to user? user maybe confused, right? 

Think about whether we can remove such a filter here.

Reply:
The frontend will not show results of internal ports; see my implementation from #3279 about what will be shown to the frontend. If an external port is materialized by the scheduler but not specified by the user, we will show the results to the user. In the next PR related to updating the frontend I will use an icon different from the eye icon to show such info.